### ISSUE TITLE
Control-Click to open a link in a new window

### ISSUE THREAD

**Issue by charles-esterbrook** [zim-desktop-wiki/zim-desktop-wiki#1623]
In Zim, you can right click a link and choose "Open in New Window" from the menu. This feature request is to add Control-Click to do the same thing.

Note that I'm not referring to the "Open in New Window" in the File menu, or the one in the accelmap:

```
(gtk_accel_path "<Actions>/UIActions/open_new_window" "<Primary>Return")
```

Both of those open up the *current* page in a new window. Instead, I'm looking to open the target page of the link in a new window.

For a non-notebook link, such as https://google.com/, Control-Click would just open it (effectively a new window anyway since it will switch to your browser).

... If this is already possible somehow, please let me know. I'm on Zim Wiki 0.74.1 installed on Windows 10.

**Comment by introt** [comment]
Related: Middle mouse-click on a page in the sidebar should open a new window #1587

**Comment by charles-esterbrook** [comment]
Also, it would make sense for Control-Click to do this in Index, Tags, Backlinks and any other place that has clickable pages.

**Comment by sojusnik** [comment]
> Also, it would make sense for Control-Click to do this in Index, Tags, Backlinks and any other place that has clickable pages.

Sounds reasonable! So ctrl+left mouse-click or a single middle mouse-click should lead to the same outcome – opening a new separate page on all clickable pages.

**Comment by introt** [comment]
I agree with the above and would thus ask @sojusnik to include this in #1587 and @charles-esterbrook to close this issue as a duplicate. Thank you!
